index.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio | Abenezer Fekadu</title>
+  <link rel="stylesheet" href="./style.css">
+ <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300;400;500;700;800&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="wrapper">
+    
+    
+      <nav class="navbar">
+
+        <div class="navbar-brand">Abenezer</div>
+    
+      
+        <ul class="navbar-list">
+          <li class="navbar-item">Portofolio</li>
+          <li class="navbar-item">About</li>
+          <li class="navbar-item">Contact</li>
+          <li class="navbar-item"><img src="./img/666162.png" alt="" class="nav-icon"></li>
+        </ul>
+      <div class="navbar-toggle">
+        <i class="fa-solid fa-bars"></i>
+      </div>
+      </nav>
+      <section class="header">
+      <div class="header-text"><h1>Hey There. <span>I'm Abenezer</span> </h1>
+      </div>
+      <div class="body-text-header"><h2>I'm a Software Developer</h2>
+      </div>
+      <div class="body-text-main">
+        <p>I can help you build a product,feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+      </div>
+      <ul class="portfolio-social">
+        <li class="portfolio-social-item"><i class="fa-brands fa-github"></i></li>
+        <li class="portfolio-social-item"><i class="fa-brands fa-linkedin-in"></i></li>
+        <li class="portfolio-social-item"><i class="fa-solid fa-hand-peace"></i></li>
+        <li class="portfolio-social-item"><i class="fa-brands fa-twitter"></i></li>
+        <li class="portfolio-social-item"><i class="fa-solid fa-m"></i></li>
+      </ul>
+      </section>
```

Comment:
- [ ] There is a background image for section of your portfolio which is supposed to be added with css background-image property. At this moment, that background image is missing on your project. Kindly add the background image to your headline section.

### Your Design.
![your-design](https://user-images.githubusercontent.com/26597432/213417248-70cad581-114e-45c7-8b31-7b406ce533b7.png)

### Figma Design
![figma-design](https://user-images.githubusercontent.com/26597432/213417285-d76e3b6b-8160-4fb1-9110-c1ba5d87052b.png)

- [ ] Also, it is required to stick to the guiding template as much as possible. This includes font, color, background-color, background-image, margin, padding among others. Kindly increase the font-size of your `p` tag content and make sure you use the same font-family for the section. This should be applied to every other tags on this section.